fitbert/fitb.py

```diff
@@ -177,12 +160,37 @@ def simplify_options(self, sent: str, options: List[str]):
                 sent = sent.replace(self.mask_token, " ".join([first_word, self.mask_token]))
         return options, sent, first_word, last_word
 
-    def rank_multi(self, sent: str, options: List[str]) -> List[str]:
+    def rank(
+        self, sent: str, options: List[str], delemmatize: bool = False
+    ) -> List[str]:
+        if len(options) == 1:
+            delemmatize = True
+        if delemmatize:
+            options = self._delemmatize_options(options)
+        scored = self._get_probs_for_words(sent, options)
+        ranked = sorted(scored.items(), key=lambda x: x[1], reverse=True)
+        return list(list(zip(*ranked))[0])
+
+    def rank_multi(self, sent: str, options: List[str], delemmatize: bool = False) -> List[str]:
         ranked_options = []
         if self.is_multi_word(options):
             options, sent, first_word, last_word = self.simplify_options(sent, options)
             ranked_options = self.sentence_prob_to_rank(sent, options)
             ranked_options = [" ".join([first_word, r, last_word]).strip() for r in ranked_options]
         else:
-            ranked_options = self.rank(sent, options=options)
+            ranked_options = self.rank(sent, options, delemmatize)
         return ranked_options
+
+    def fitb(self, sent: str, options: List[str], delemmatize: bool = False) -> str:
+        ranked = self.rank_multi(sent, options, delemmatize)
+        best_word = ranked[0]
+        return sent.replace(self.mask_token, best_word)
+
+    def mask_fitb(self, sent: str, span: Tuple[int, int]) -> str:
+        masked_str, replaced = self.mask(sent, span)
+        options = [replaced]
+        return self.fitb(masked_str, options, delemmatize=True)
+
+
```

Comment:
really nit-picky, but the file should end with one blank line.

I use `black` as an auto-formatter and recommend everyone else does, it will automatically fix things like this

Reply:
This is what my `.vscode/settings.json` looks like (I install certain things like linters and formatters with anaconda, so I don't have to install them in every env):

```json
{
    "python.pythonPath": "venv/bin/python",
    "python.linting.enabled": true,
    "python.linting.mypyEnabled": true,
    "python.linting.mypyPath": "/anaconda3/bin/mypy",
    "python.linting.mypyArgs": [
        "--ignore-missing-imports",
        "--namespace-packages",
        "--follow-imports=silent",
        "--show-column-numbers"
    ],
    "python.linting.pylintEnabled": false,
    "python.formatting.provider": "black",
    "python.formatting.blackPath": "/anaconda3/bin/black",
    "files.exclude": {
        "**/.git": true,
        "**/.svn": true,
        "**/.hg": true,
        "**/CVS": true,
        "**/.DS_Store": true,
        "**/*.pyc": true,
        "**/__pycache__": true,
        "**/.mypy_cache": true
    }
```

Reply:
Good point! I forgot to run black here. Thx!

Reply:
Done.